assets/app/scripts/filters/resources.js

```diff
@@ -779,16 +779,8 @@ angular.module('openshiftConsole')
       }
     };
   })
-  .filter('humanizeKind', function () {
-    return function(kind) {
-      if (!kind) {
-        return kind;
-      }
-
-      // ReplicationController -> Replication Controller
-      // https://lodash.com/docs#startCase
-      return _.startCase(kind);
-    };
+  .filter('humanizeKind', function (startCaseFilter) {
```

Comment:
ok we can leave it, doesnt hurt anything

On Thu, Mar 17, 2016 at 1:32 PM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/scripts/filters/resources.js
> <https://github.com/openshift/origin/pull/8095#discussion_r56544854>:
>
> > @@ -779,16 +779,8 @@ angular.module('openshiftConsole')
> >        }
> >      };
> >    })
> > -  .filter('humanizeKind', function () {
> > -    return function(kind) {
> > -      if (!kind) {
> > -        return kind;
> > -      }
> > -
> > -      // ReplicationController -> Replication Controller
> > -      // https://lodash.com/docs#startCase
> > -      return _.startCase(kind);
> > -    };
> > +  .filter('humanizeKind', function (startCaseFilter) {
>
> Or change the case to lower case? We're inconsistent about that today.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8095/files/e63659923bebc75526886378f48427f234695496#r56544854>
>